core/client/fs/src/main/java/alluxio/client/file/FileInStream.java

```diff
@@ -66,13 +66,13 @@
 public class FileInStream extends InputStream implements BoundedStream, PositionedReadable,
     Seekable {
   private static final Logger LOG = LoggerFactory.getLogger(FileInStream.class);
-  private static final int MAX_WORKERS_TO_RETRY =
-      Configuration.getInt(PropertyKey.USER_BLOCK_WORKER_CLIENT_READ_RETRY);
 
+  private final int mBlockWorkerClientReadRetry;
```

Comment:
I like this pattern of basing field names off of the configuration property name